src/openapiGenerator/mockGenerator/formatResponse.ts

```diff
@@ -0,0 +1,41 @@
+// import { OpenAPIObject, PathItemObject } from "openapi3-ts";
+
+import { generateData, generateModelsWithData } from "./generateData";
+
+export const APPLICATION_JSON = "application/json";
+
+export type ResponsesType = {
+    [path: string]: {
+        [APPLICATION_JSON]: { schema: any };
+    };
+};
+
+export const extractResponses = (obj: any): ResponsesType => {
+    const extracted: any = {};
+    const modelSchemas = generateModelsWithData(obj);
+    Object.keys(obj.paths).forEach(path => {
+        const methods: any = obj.paths[path];
+        Object.keys(methods).forEach((method: string) => {
+            const api: any = methods[method];
+            const { responses, operationId } = api;
```

Comment:
how did you know there is `responses` and `operationId` inside `api`? if there is a predefined type then please put it in place instead of defining `api` as `any`. If there is not type but the api will always look the same, then you need to define your own interface and use it.

Reply:
added